recipes/pretext-suite/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: pretext-suite
+  version: "0.0.1"
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - pretextmap
+    - pretextsnapshot
```

Comment:
Would it makes sense to add version constrains to these packages? Just trying to think ahead here, such that, e.g.,
`pretext-suite 0.0.1` would require `pretextsnapshot <0.1.0a0` and a future `pretext-suite 0.1.0` would require `pretextsnapshot >=0.1,<0.2.0a0`.

Reply:
not really, not right now, the two requirments are a version number appart already,

I can add contraints in the future if i ever make a major change to one of the packages